aidies-project/client/src/components/Navigation.js

```diff
@@ -65,10 +67,19 @@ function Navigation(props) {
     const dispatch = useDispatch();
     const [keyword, setKeyword] = useState('');
     const history = useHistory();
-    
+
+    const [selected, setSelected] = useState(false);
+    const categories = useSelector(state => state.campaign.categories);
+
+    useEffect(()=>{
+        dispatch(getCategories());
+    }, [])
+
+    const handleToggle = e => {
+        
+    }
```

Comment:
I think you're still working on this, but just a remote note about this empty function!